ui_test/src/lib.rs

```diff
@@ -230,6 +235,11 @@ fn run_test(
     }
     let output = miri.output().expect("could not execute miri");
     let mut errors = config.mode.ok(output.status);
+    // Always remove annotation comments from stderr.
+    let annotations = Regex::new(r"\s*//~.*").unwrap();
```

Comment:
Ah, moving this hard-coded normalization here means the test suite does not use it any more... :/

Reply:
Now I am already working on factoring this a bit more so we can test it better. ;)

Reply:
compiletest hard-codes the `//~` normalization as well, right? So why would we remove it?

Reply:
ah ok ^^ I thought just to get the PR to pass quicker

Reply:
So are you find landing this the way it looks now?

Reply:
Ah, I didn't see your other PR, could have resolved this after yours... anyway, thanks!

Reply:
🤷 was quicker to resolve the conflicts on my end.